tests/core/tools/style.js

```diff
@@ -168,6 +168,10 @@
 
 		'test style.parse.border with style and hsla color': function() {
 			objectAssert.areEqual( { style: 'dotted', color: 'hsla(10,30%,30%,1)' }, CKEDITOR.tools.style.parse.border( 'dotted hsla(10,30%,30%,1)' ) );
+		},
+
+		'test style.parse.border with color white spaces': function() {
+			objectAssert.areEqual( { style: 'solid', color: 'rgba(10,20,30,.75)', width: '1px' }, CKEDITOR.tools.style.parse.border( '1px solid rgba(10,  20, 30,  .75)' ) );
```

Comment:
I'd rather like to have original `color` syntax, in a sense that spaces are not being removed if possible.

Reason for this is that by doing so we reduce possibility of CKEditor changing existing value format.

Reply:
You can do achieve that in `CKEDITOR.tools.style.parse.border` easily by obtaining `ret.color` before `forEach`.